packages/graphql/tests/integration/enums.int.test.ts

```diff
@@ -87,4 +87,58 @@ describe("enums", () => {
             await session.close();
         }
     });
+
+    test("should create a movie (with a default enum)", async () => {
+        const session = driver.session();
+
+        const statusResolver = {
+            ACTIVE: "active",
+        };
+
+        const typeDefs = `
+            enum Status {
+                ACTIVE
+            }
```

Comment:
Just to make this more realistic, I would add one or two extra options to this status
```suggestion
            enum Status {
                ACTIVE
                INACTIVE
                EATING
            }
```